tests/h/admin/views/features_test.py

```diff
@@ -75,21 +77,58 @@ def test_cohorts_index_when_no_cohorts():
     assert result["results"] == []
 
 
-def test_new_cohort_creates_cohort():
+def test_new_cohort_creates_cohort_with_no_members():
     req = DummyRequest(db=db.Session)
     req.params['add'] = 'cohort'
     result = views.cohorts_add(req)
     assert isinstance(result, httpexceptions.HTTPSeeOther)
 
     result = views.cohorts_index({}, req)
     assert len(result["results"]) == 1
-    assert result["results"][0].name == "cohort"
+
+    cohort = result["results"][0]
+    assert cohort.name == "cohort"
+    assert len(cohort.members) == 0
+
+
+def create_cohort(req):
+    req.params['add'] = 'cohort'
+    views.cohorts_add(req)
+    return views.cohorts_index({}, req)['results'][0]
+
+
+def add_user_to_cohort(req, user, cohort):
```

Comment:
Ditto. You could achieve the same database state in about as many lines by just creating a `FeatureCohort` object yourself and adding it to the session.